### ISSUE TITLE
[Resource] Demonology

### ISSUE THREAD

**Issue by warpdragon** [AngelArenaAllstars/suggestions#29]
Written Demonology Reference
[The Lesser Key of Solomon - Goetia; Translated by S. L. Mathers](http://www.hermetics.org/pdf/grimoire/goetia.pdf)
Graphical index of the 72 Demons
![image](https://cloud.githubusercontent.com/assets/13878439/22187005/a9f6c9c8-e0b3-11e6-9c3b-3d03d0fa32c9.png)

**Comment by yommi1999** [comment]
Awesome. So tons of bosses will have the visual appearance of these guys?

**Comment by warpdragon** [comment]
@yommi1999 I don't think so. I think the Art team will go whatever direction they think is best based on a number of factors including personal style and the project's artistic direction. 

I just posted it because I find it interesting and I think it's relevent to the project.

**Comment by yommi1999** [comment]
Ah alright so waifus will still be there then 😀.

Op 30 jan. 2017 21:12 schreef "warpdragon" <notifications@github.com>:
